sachet/sachet.default

```diff
@@ -0,0 +1 @@
+SACHET_OPTS=-config /etc/sachet.yml
```

Comment:
What do you think about having the configuration file located at `/etc/prometheus/sachet.yml` by default?

For example, alertmanager's config is `/etc/prometheus/alertmanager.yml` and blackbox_exporter's config is `/etc/prometheus/blackbox.yml`.